UMN-AutoPackager/public/Get-GlobalConfig.ps1

```diff
@@ -0,0 +1,30 @@
+function Get-Config {
```

Comment:
This should be made Get-GlobalConfig or Get-ModuleConfig (maybe Get-UMNAutopackagerConfig to be unique?)

Reply:
renamed the functions to Get-UMNGlobalConfig and Set-UMNGlobalConfig